packages/styles/src/env.d.ts

```diff
@@ -0,0 +1,7 @@
+// CSS modules
```

Comment:
Should we name this file `CSSModulesClasses.d.ts`?

Reply:
Going with `env.d.ts` allows us adding other style related globals here in the future.

The idea is: you reference `bf-wc-styles/env` in your typescript env file and you get the environment supporting CSS modules.